kerastuner/tuners/multi_execution_tuner.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2019 The Keras Tuner Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"Tuner that runs multiple executions per Trial."
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from ..engine import trial as trial_lib
+from ..engine import tuner as tuner_module
+from ..engine import oracle as oracle_module
+from ..engine import hyperparameters as hp_module
+from ..abstractions.tensorflow import TENSORFLOW_UTILS as tf_utils
+
+import collections
+import copy
+import json
+import numpy as np
+import random
+from tensorflow import keras
+
+
+class MultiExecutionTuner(tuner_module.Tuner):
```

Comment:
It seems like this class would belong in the `engine` and not in `tuners` (which is for end user-facing specialized tuners)

Reply:
Done